src/cmd/completion.go

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The UDS Authors
+
+// Package cmd contains the CLI commands for UDS.
+package cmd
+
+import (
+	"os"
+
+	"github.com/defenseunicorns/uds-cli/src/config/lang"
+	"github.com/spf13/cobra"
+)
+
```

Comment:
```suggestion

// We manually create the completion cmds because autogen'ing the docs includes Powershell completions,
// which we don't support. Cobra doesn't provide a neat mechanism for excluding just Powershell so if we want
// these completion cmds built into the CLI we have to create them manually
```

Reply:
^ this may not come out right if you hit "commit suggestion", might want to copy/paste as you see fit into your branch